recipe-client-addon/test/browser/browser_about_studies.js

```diff
@@ -0,0 +1,147 @@
+"use strict";
+
+Cu.import("resource://shield-recipe-client/content/AboutPages.jsm", this);
+
+function withAboutStudies(testFunc) {
+  return async function inner(...args) {
+    return BrowserTestUtils.withNewTab(
+      {gBrowser, url: "about:studies"},
```

Comment:
protip: you can just pass the url string instead of the object, and it'll just use the topmost browser window's gBrowser.